web/react/components/filtered_user_list.jsx

```diff
@@ -48,6 +92,7 @@ class FilteredUserList extends React.Component {
 
         let users = this.props.users;
 
+        // this logic might be better placed in UserStore
```

Comment:
Yeah, I think the higher-up, parent components just want to pull their data from the store directly, but any sort of derived data later on would be computed from that. I'm thinking sort of along the lines of [this](https://facebook.github.io/react/tips/props-in-getInitialState-as-anti-pattern.html)